package.json

```diff
@@ -18,15 +18,7 @@
     "test-coverage": "nyc npm run test-unit",
     "test-unit": "mocha --recursive test/unit/ --exit",
     "test-integration": "KNEX_CONFIG=testing mocha --recursive test/integration/ --timeout 15000 --exit",
-    "test-e2e-cypress": "KNEX_CONFIG=testing cypress run --config video=false",
-    "test-e2e": "gulp --gulpfile gulpfile-e2e.js",
-    "test-e2e-smoke": "gulp --gulpfile gulpfile-e2e.js e2e-smoke",
-    "test-e2e-ie8": "wdio test/wdio.conf.sauce.ie8.js",
-    "test-e2e-firefox": "wdio test/wdio.conf.sauce.firefox.js",
-    "test-e2e-ie8-jenkins": "wdio test/wdio.conf.sauce.ie8.jenkins.js",
-    "test-e2e-firefox-jenkins": "wdio test/wdio.conf.sauce.firefox.jenkins.js",
-    "test-e2e-chrome-jenkins": "wdio test/wdio.conf.sauce.chrome.jenkins.js",
-    "test-e2e-chrome-jenkins-smoke": "wdio test/wdio.conf.sauce.chrome.jenkins.smoke.js",
+    "test-e2e": "KNEX_CONFIG=testing cypress run --config video=false",
```

Comment:
Is it worth adding a line `"test-e2e-ui": "KNEX_CONFIG=testing cypress open",` too, like HwPV external has, so it's quick to get the Cypress UI open?